lib/tapioca/dsl/compilers/protobuf.rb

```diff
@@ -206,7 +206,7 @@ def type_of(descriptor)
             # > field, even if it was not defined in the enum.
             "T.any(Symbol, Integer)"
           when :message
-            descriptor.subtype.msgclass.name
+            descriptor.subtype.msgclass.name || "T.untyped"
```

Comment:
No, that returns the name of the protobuf message, not the class name; I was wrong about that in the previous change.

For example, in the binary serialized test, if we hadn't assigned the `MyCart.Progress` message to a constant, its `subtype.msgclass.name` would be `nil`, but it would return `MyCart.Progress` from `submsg_name`, which is not a valid constant name.

Reply:
No, it wouldn't since there is no named constant defined in Ruby that corresponds to that `msgclass`. That constant is anonymous, so we can't refer to it.